authConfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "domain": "dev-lbeeue481zqdew5i.us.auth0.com",
+  "clientId": "H8LpvyDBdLX2LsdoGz9R4T3qY2dBSVe3",
```

Comment:
Is this the auth0 credentials/token? clientId refers to the auth0 account user, correct?

Reply:
these are the credentials for the auth0 environment we use